docs/sdk/api/syncActions.md

```diff
@@ -432,3 +435,50 @@ client.execute(zonesRequests)
 .then(result => ...)
 .catch(error => ...)
 ```
+
+## `createSyncProductTypes(actionGroups)`
+
+> From package [@commercetools/sync-actions](/sdk/api/README.md#sync-actions).
+
+Creates a [sync action](/sdk/Glossary.md#sync-action) that allows to build API update actions for _zones_.
```

Comment:
> update actions for _zones_

Zones? Shouldn't it be `ProductTypes`?

Reply:
cmd + shift + d (duplicate) and missed to replace this :P